docs/reference/limitations.rst

```diff
@@ -31,16 +31,13 @@ Shared libraries on Windows
 On Windows, ``meson-python`` cannot encapsulate shared libraries
 installed as part of the Meson project into the Python wheel for
 Python extension modules or executables, in a way suitable for them to
-be found at run-time.
-
-This limitation can be overcome with static linking or using
-`delvewheel`_ to post-process the Python wheel to bundle the required
-shared libraries and include the setup code to properly set the
-library search path.
+be found at run-time without cooperation from the Python package, see
+the documentation for the
+:option:`tool.meson-python.allow-windows-internal-shared-libs` option
+and the :ref:`shared-libraries` guide.
```

Comment:
I'm wondering whether it makes sense to delete this section completely - it's not actually a limitation anymore after this PR, it works with a few lines of code, and in case someone runs into it when forgetting the opt-in step, the error message points at the thing to do.

Reply:
Agreed. I'll remove it